### PR TITLE
[skip ci] Readme: add client install, segsadmin config and some notes

### PR DESCRIPTION
**Pull request by valcryst**
## Summary
added client installation guide.
how to configure with segsadmin (linux/windows)
add note about intel/amd problems
add ingame slash command list with access level

### CODE REVIEW

File: docs/COH-CLIENT_INSTALL-WINDOWS.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+   > Your line should end something like this: `.../CoX.exe" -project SEGS`.  
+   Please note the space between the quotation mark and the dash.  
+   
+   If you would like to run the client in windowed mode, you may also add these optional command line arguments:  
+
+   `-fullscreen 0 -screen 1024 768`.  
+   
+   When you want to launch the client, you will need to use this shortcut.
+
+   NOTE: This is a known issue with Intel or AMD/Radeon graphics cards.
```
Comment (broxen) [inline docs/COH-CLIENT_INSTALL-WINDOWS.md]: Let's be more specific about what the known issue is. Maybe make this entire section:

`4. **Common Issues and Troubleshooting**`

You can mention the discord for additional topics not covered.

File: docs/SEGS-SLASH_COMMANDS.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
+| Command                        | Description                                                                    | Access Level |
```
Comment (broxen) [inline docs/SEGS-SLASH_COMMANDS.md]: We might point users to the wiki and create a wiki page for this? Or have them use `/cmdlist` in-game, which will show all commands that you have access to.

This way we won't have to constantly update this list as we continue to add slashcommands.

File: docs/COH-CLIENT_INSTALL-WINDOWS.md (hunk 1 above)
Comment (valcryst) [reply docs/COH-CLIENT_INSTALL-WINDOWS.md]: How about an extra page for this, where also current github issues related to this problems get linked?
Then we could point from every "guide" to this page.

File: docs/SEGS-SLASH_COMMANDS.md (hunk 2 above)
Comment (valcryst) [reply docs/SEGS-SLASH_COMMANDS.md]: I would realy like to have it "offline" too, so i would then create a wiki page for this.
For me as new user it was not easy to gather all information i wanted - this was one of them :)
So atleast a link from the install instructions to a page where informations about possible
commands/options for the server can be viewed.

File: docs/COH-CLIENT_INSTALL-WINDOWS.md (hunk 1 above)
Comment (broxen) [reply docs/COH-CLIENT_INSTALL-WINDOWS.md]: Honestly, I'd recommend only mentioning the Shader issue, linking to our Issues page on GitHub and our discord for chat.

What I'd like to avoid is another document that will require constant updates. Especially given that our resources are already limited.

Also, I feel that if users have to jump around too many files, they'll stop reading altogether. I'd prefer to get then the pertinent information as quickly as possible.

File: docs/SEGS-SLASH_COMMANDS.md (hunk 2 above)
Comment (broxen) [reply docs/SEGS-SLASH_COMMANDS.md]: Makes sense. I'd at least mention the `/cmdlist` at the top of the wiki, if not in the doc itself.

As with my other comment, I want to minimize the number of places we have to update information. If we get too busy to keep it updated it becomes stale and less useful.

File: docs/COH-CLIENT_INSTALL-UBUNTU.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
+CoH Client Installation on Ubuntu 18.04 using PlayOnLinux  
+------  
+  
+1.**Install CoH Client version 0.22**  
+  
+SEGS will only communicate with the CoH client from Issue 0 (version 0.22).  
+This can be found through various sources online, or via the following Torrent-magnet link provided the City of Heroes community:
```
Comment (broxen) [inline docs/COH-CLIENT_INSTALL-UBUNTU.md]: Small typo "provided **by** the City of Heroes Community"

File: docs/COH-CLIENT_INSTALL-WINDOWS.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
+CoH Client Installation
+------
+
+**1. Install CoH Client version 0.22**
+
+SEGS will only communicate with the CoH client from Issue 0 (version 0.22).  
+This can be found through various sources online, or via the following magnet link provided the City of Heroes community:
```
Comment (broxen) [inline docs/COH-CLIENT_INSTALL-WINDOWS.md]: Same typo here: missing `by`

File: docs/COH-CLIENT_INSTALL-WINDOWS.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+The client version we use is not capable of handling those graphics cards.  
+
+https://github.com/Segs/Segs/issues/736  
+
+For Help with installation and configuration of your local SEGS, please see visit us on discord.  
+A patch is in the works but we need a skilled debugger that's capable of testing/debugging on an  
+Intel or Radeon card. If you're experienced in debugging on Windows we would be pleased if you contact us.  
+
+**Chat with us on Discord!**  
+We're currently hosting a `#segs` channel on Splat Chat. You can join by clicking this link:
```
Comment (broxen) [inline docs/COH-CLIENT_INSTALL-WINDOWS.md]: This is no longer accurate and we need to change everywhere we use this verbiage.

Now "Splat Chat" has become the official SEGS discord server. Let's rephrase this to that effect, and point people to #pocket-d for general chat or #troubleshooting for help.

File: docs/COH-CLIENT_INSTALL-WINDOWS.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+We're currently hosting a `#segs` channel on Splat Chat. You can join by clicking this link:  
+
+ [**Click here to join SEGS Discord**](https://discord.segs.io/)  
+
+Or if **IRC** chat is more your speed, you can join us on IRC:  
+
+  **Server:** irc.segs.io  
+  **Port:** 6667  
+  **SSL port:** 6697  
+  **Channel:** #segs
```
Comment (broxen) [inline docs/COH-CLIENT_INSTALL-WINDOWS.md]: We might need to change this channel name. I'm checking with @ldilley

File: docs/COH-CLIENT_INSTALL-UBUNTU.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+The client version we use is not capable of handling those graphics cards.  
+  
+https://github.com/Segs/Segs/issues/736  
+  
+For Help with installation and configuration of your local SEGS, please see visit us on discord.  
+A patch is in the works but we need a skilled debugger that's capable of testing/debugging on an  
+Intel or Radeon card. If you're experienced in debugging on Windows we would be pleased if you contact us.  
+  
+**Chat with us on Discord!**  
+We're currently hosting a `#segs` channel on Splat Chat. You can join by clicking this link:
```
Comment (broxen) [inline docs/COH-CLIENT_INSTALL-UBUNTU.md]: See my other note about this section

File: docs/SEGS-CONFIGURE_WITH_SEGSADMIN.md [hunk 8]
```diff
@@ -0,0 +1,79 @@
+Configure SEGS using SEGSADMIN
```
Comment (broxen) [inline docs/SEGS-CONFIGURE_WITH_SEGSADMIN.md]: This file duplicates the text elsewhere. I'm worried about the maintenance. What's the goal with this file?

If it's to reduce duplication, then let's link to this from the `docs/SEGS-COMPILE_AND_INSTALL-WINDOWS.md` otherwise maybe we don't need this separate file after all

File: docs/COH-CLIENT_INSTALL-WINDOWS.md (hunk 6 above)
Comment (ldilley) [reply docs/COH-CLIENT_INSTALL-WINDOWS.md]: This information is still valid.

File: docs/CONTRIBUTING.md [hunk 9]
```diff
@@ -43,7 +43,8 @@ We have an official message board and a FAQ where the community chimes in with h
 
 ### Chat with us on Discord
 
-Chat with us on Discord! We're currently hosting a `#segs` channel on Splat Chat. You can join by clicking this link:
+Come and join us in `#pocket-d` channel for general chat or `#troubleshoothing` for support.
```
Comment (broxen) [inline docs/CONTRIBUTING.md]: Small typo in `troubleshooting`

File: docs/COH-CLIENT_INSTALL-WINDOWS.md [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+
+https://github.com/Segs/Segs/issues/736
+
+For Help with installation and configuration of your local SEGS, please see visit us on discord.
+A patch is in the works but we need a skilled debugger that's capable of testing/debugging on an
+Intel or Radeon card. If you're experienced in debugging on Windows we would be pleased if you contact us.
+
+**Chat with us on Discord!**
+
+Come and join us in `#pocket-d` channel for general chat or `#troubleshoothing` for support.
```
Comment (broxen) [inline docs/COH-CLIENT_INSTALL-WINDOWS.md]: Same typo in `troubleshooting`
